rocky/tests/conftest.py

```diff
@@ -450,8 +451,12 @@ def mock_scheduler(mocker):
     return mocker.patch("rocky.views.ooi_detail.scheduler.client")
 
 
-def get_boefjes_data():
-    return json.loads((Path(__file__).parent / "stubs" / "katalogus_boefjes.json").read_text())
+def get_stub_path(file_name: str) -> Path:
+    return Path(__file__).parent / "stubs" / file_name
```

Comment:
Might as well be a static variable or inlined in the function below

Reply:
I'm using it elsewhere as well though